src/main/java/ch/njol/skript/SkriptCommand.java

```diff
@@ -146,6 +147,17 @@ public boolean onCommand(CommandSender sender, Command command, String label, St
 						});
 				}
 
+				else if (args[1].equalsIgnoreCase("changed")) {
+					reloading(sender, "scripts that have changed since last load");
```

Comment:
you need a lang entry for the string. it would go with one of the others here:
https://github.com/SkriptLang/Skript/blob/4d4094486fd0fc21771f86e30b20104262f6b5f1/src/main/resources/lang/english.lang#L65

it might be better to place it at `skript command.reload.changed since last load` and use 
`reloading(sender, "changed since last load");`